New version: TensorOperations v5.3.2

JuliaRegistrator opened the pull request:
- Registering package: TensorOperations
- Repository: https://github.com/QuantumKitHub/TensorOperations.jl
- Created by: @Jutho
- Version: v5.3.2
- Commit: f1fd025065cc8b97b01919ba3f68c584f4927006
- Reviewed by: @Jutho
- Reference: https://github.com/QuantumKitHub/TensorOperations.jl/commit/f1fd025065cc8b97b01919ba3f68c584f4927006#commitcomment-173814684
- Description: Julia package for tensor contractions and related operations
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd67b342a571093725a1278c72cce6085d4e1d82cddc27ddcb9b7ee87377e4743e5767277f49b6d84fd1ffa8eca15534f5055e7455af5995573514ed49a52fc161f3c7d69a75bb3fc782012acf20f57d4b9fca49ba754aba43b2c46a2208a68954b56656e6c2d4f5a2ea94d72382387e194484a784f289dbe8cb780b1d91effb72cca73802954e02dba186d3a5db63c4edb4f65d871e5f42964480d1788c607e805df914f19986a23c05dd1fa7a78f3b80601f7c34a95b3fb22a5f8ad2a1d161b59 -->

Jutho commented:
@kshyatt , would you have time to to take a look at what the proper cuTensor compat entry should be?

JuliaRegistrator committed:
New version: TensorOperations v5.3.2

UUID: 6aa20fa7-93e2-5fca-9bc0-fbd0db3c71a2
Repo: https://github.com/QuantumKitHub/TensorOperations.jl.git
Tree: 0f3b9bf5ad0e4d3c79a89bb8a5e13c3443c69088

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

Jutho commented:
Ok. It could also be "2.1" or "2.1.1". I think it is the ">=2.1.1" that is problematic, as that also includes "3", "4" and any future version.
